app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx

```diff
@@ -143,7 +131,7 @@ const inviteFormSelector = formValueSelector('inviteUserForm');
   },
 )
 @reduxForm({
-  form: 'inviteUserForm',
+  form: INVITE_USER_FORM,
```

Comment:
Don't we need to use the new autocomplete field in the modal?
I see this on designs, but perhaps it is beyond the scope of this task.

Reply:
This requires another task